### PR TITLE
chore(Azure): remove app-level logging to Azure Monitor

### PR DESCRIPTION
**Pull request by afeld**
Wasn't working anyway. Alternative to https://github.com/cal-itp/benefits/pull/1260. Though this _could_ be merged independently of https://github.com/cal-itp/benefits/pull/1259, we would likely only want to do so together so that we still have a system for error monitoring.

### CODE REVIEW

**Comment by afeld** [comment]
Moving this to Draft state until we discuss at [the next dev meeting](https://docs.google.com/document/d/1vActZALSbRwO0HxDvm-YQK37p22SPbL4XuMdcRx-BI8/edit#heading=h.9ra1yb7sizl2).

**Comment by afeld** [comment]
Following [the discussion at the dev workshop](https://docs.google.com/document/d/1vActZALSbRwO0HxDvm-YQK37p22SPbL4XuMdcRx-BI8/edit#bookmark=id.rl1i47496vbt), determined logging to Azure Monitor wasn't worth the hassle. Expanded this pull request to remove the integration. Also updated [the doc with the direct links](https://docs.google.com/document/d/11EPDIROBvg7cRtU2V42c6VBxcW_o8HhcyORALNtL_XY/edit#).

File: benefits/settings.py [hunk 1]
```diff
@@ -226,7 +203,33 @@ def _filter_empty(ls):
...
+        "handlers": ["console"],
+        "level": LOG_LEVEL,
+    },
+    "loggers": {
+        "django": {
+            "handlers": ["console"],
+            "propagate": False,
+        },
+    },
+}
```
Comment (afeld) [inline benefits/settings.py:232]: No change, just moved back into this file.

**Review by thekaveman — APPROVED** [review]
👍

**Pull request merged**